src/screens/Home.tsx

```diff
@@ -11,30 +13,9 @@ import {
 import { Product } from "@app/interfaces/products";
 import { ProductContext } from "@app/context/product";
 import { useIsFocused } from "@react-navigation/native";
-const categories = ["fruits", "vegetables", "snacks", "meat", "bakery"];
-const productsList = [
-  {
-    id: 0,
-    name: "Tomato",
-    origin: "Spain",
-    image: "Spain",
-    price: 30.0,
-  },
-  {
-    id: 1,
-    image: "Potato",
-    name: "Potato",
-    origin: "Portugal",
-    price: 20.0,
-  },
-  {
-    id: 2,
-    image: "Mandioquinha",
-    name: "Mandioquinha",
-    origin: "Brazil",
-    price: 50.0,
-  },
-];
+
+const categories = ["fruits", "vegetables", "fish", "meat", "bakery"];
```

Comment:
eu moveria isso para um ENUM pq ai voce pode usar

```suggestion
enum Categories {
  Fruits,
  Vegetables,
  Fish,
  Meat,
  Bakery
}
```

Reply:
As categorias vão vir do CMS. Ainda não implementei isso mas faz sentido sim  👍